src/Engine/Objects/SpriteObject.cpp

```diff
@@ -1419,27 +1419,27 @@ bool _46BFFA_update_spell_fx(unsigned int uLayingItemID, int pid) {
             //        - 1] + 1; pAudioPlayer->PlaySound((SoundID)v125, v124, 0,
             //        -1, 0, v97, 0, 0);
             pAudioPlayer->PlaySpellSound(pSpriteObjects[uLayingItemID].spell_id,
-                                         PID(OBJECT_Item, uLayingItemID));
+                                         PID(OBJECT_Item, uLayingItemID), true);
             return 0;
         }
 
         case SPRITE_SPELL_MIND_CHARM:
         case SPRITE_SPELL_LIGHT_PARALYZE:
-        case SPRITE_SPELL_DARK_SHRINKING_RAY: {
-            int v143 = 17030;
-            switch (pSpriteObjects[uLayingItemID].uType) {
-                case SPRITE_SPELL_MIND_CHARM:
-                    v143 = 15040;
-                    break;
-                case SPRITE_SPELL_EARTH_SLOW:
-                    v143 = 13010;
-                    break;
-                case SPRITE_SPELL_DARK_SHRINKING_RAY:
-                    v143 = 18030;
-                    break;
-                default:
-                    break;
-            }
+        case SPRITE_SPELL_DARK_SHRINKING_RAY:  {
+            // int v143 = 17030;
+            // switch (pSpriteObjects[uLayingItemID].uType) {
+            //     case SPRITE_SPELL_MIND_CHARM:
+            //         v143 = 15040;
+            //         break;
+            //     case SPRITE_SPELL_EARTH_SLOW:
+            //         v143 = 13010;
+            //         break;
+            //     case SPRITE_SPELL_DARK_SHRINKING_RAY:
+            //         v143 = 18030;
+            //         break;
+            //     default:
+            //         break;
+            // }
```

Comment:
delete me :)

Reply:
I decided to check these sound IDs and found out they are not correspond with sound IDs in spell table but correspond to some other random spells.
I'll compare to vanilla and see what's the deal.